broker/api.go

```diff
@@ -10,9 +10,16 @@ type PermittedAccount struct {
 	PermittedRoleName []string
 }
 
+type AWSCredentialsJSON struct {
```

Comment:
Please remove AWS-specific code and types from this package. I suggest a `TokenWriter` interface with a single `WriteToken` method which looks the same as the standard library `Write` method for the `io.Writer` interface.